hack/tools/go.mod

```diff
@@ -3,23 +3,10 @@ module sigs.k8s.io/cluster-api/hack/tools
 go 1.16
 
 require (
-	github.com/golang/mock v1.4.3
-	github.com/golangci/golangci-lint v1.17.1
+	github.com/golangci/golangci-lint v1.37.1
+	github.com/jteeuwen/go-bindata v3.0.7+incompatible
 	k8s.io/code-generator v0.17.4
 	sigs.k8s.io/controller-tools v0.2.8
+	sigs.k8s.io/kustomize/kustomize/v3 v3.10.0
 	sigs.k8s.io/testing_frameworks v0.1.1
 )
-
-replace (
-	// TODO(vincepri) Remove the following replace directives, needed for golangci-lint until
```

Comment:
removing it: [golangci/golangci-lint#595](https://github.com/golangci/golangci-lint/issues/595) is fixed.